### PR TITLE
native browser context menu disenabled on file explore

### PR DESCRIPTION
**Pull request by iamsethsamuel**
…1 fixed
https://github.com/ethereum/remix-project/issues/2907#issue-1371404341 issue fixed

### CODE REVIEW

**Comment by ioedeveloper** [comment]
Give your PR a better title.

**Comment by iamsethsamuel** [comment]
> Give your PR a better title.

I just did that

**Review by joeizang — CHANGES_REQUESTED** [review]
Why do you have a package-lock.json? We use yarn and it seems you have added a package-lock.json file which would mean the use of npm instead of yarn.

**Review by joeizang — COMMENTED** [review]
You will need to rebase and update your PR

**Comment by iamsethsamuel** [comment]
> You will need to rebase and update your PR

Okay sir

**Review by ioedeveloper — CHANGES_REQUESTED** [review]
Change implementation and logic to use `contextMenuActions` type `'workspace'`

File: libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx [hunk 1]
```diff
@@ -830,8 +856,13 @@ export function Workspace () {
             </div>
           </div>
         }
+        {showContextMenu && <FileExplorerRootMenu focusContext={focusContext}
```
Comment (ioedeveloper) [inline libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx:859]: Second context menu is not needed.

**Review by ioedeveloper — CHANGES_REQUESTED** [review]

File: libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx (hunk 1 above)
Comment (iamsethsamuel) [reply libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx:859]: Okay, so how should I show the context menu?

Comment (ioedeveloper) [reply libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx:859]: See gitter comments.

**Pull request closed without merging**